static/LocationsData.json

```diff
@@ -2,6 +2,7 @@
     "parents": {
         "LOCATION_PARENT_ICA_FACILITY": {
             "Id": "LOCATION_PARENT_ICA_FACILITY",
+            "Opportunities": 4,
```

Comment:
Seems dangerous to hard-code this, perhaps we should check all contracts present on that location?

Reply:
This is the data merged from `static/Destinations.json`, and I think we are safe enough to use it. I haven't seen any mods that attempt at adding mission stories, and I doubt we will get new missions from IOI that add mission stories (even Ambrose does not have any).